automatic/waterfox/update.ps1

```diff
@@ -25,23 +31,40 @@ function global:au_SearchReplace {
       "(?i)(^\s*softwareName\s*=\s*)'.*'" = "`${1}'$softwareName'"
       "(?i)(^\s*packageName\s*=\s*)'.*'"  = "`${1}'$($Latest.PackageName)'"
     }
+    ".\waterfox.nuspec"  = @{
+      "(?i)(^\s*\<id\>).*(\<\/id\>)" = "`${1}$($Latest.PackageName)`${2}"
+      "(?i)(^\s*\<title\>).*(\<\/title\>)" = "`${1}$($Latest.Title)`${2}"
+    }
   }
 }
 
-function global:au_GetLatest {
+function Get-Waterfox {
+param(
+    [string]$build
+)
   $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
 
-  $re    = '(?:Waterfox)%20([\d]{0,2}[\.][\d]{0,2}[\.][\d]{0,2})(%20Setup\-\d+\.[ex]+)|(?:Waterfox)%20([\d]{0,2}[\.][\d]{0,2}[\.][\d]{0,2})(%20Setup)(?!\-\d+)(\.[ex]+)'
+  $re    = "(?:Waterfox%20$build%20)([\d]{0,4}[\.][\d]{0,2})(%20Setup\.[ex]+)|(?:Waterfox%20$build%20)([\d]{0,4}[\.][\d]{0,2}[\.][\d]{0,2})(%20Setup\.[ex]+)"
   $url   = $download_page.links | ? href -match $re | select -First 1 -expand href
   if (!$url) {
     $re = 'Setup\.exe$' # If we didn't get a url with the previous regex, we use a much simpler way
     $url = $download_page.links | ? href -match $re | select -First 1 -expand href
   }
 
   $version  = $url -split '%20| ' | select -Last 1 -Skip 1
-
+  if ($build -eq 'Classic') { $build = 'Classic'; $dash = '-' } else { $build=$dash = '' }
   # We need to replace the space in the url, otherwise we'll get an invalid url error.
-  return @{ URL64 = ($url -replace ' ','%20'); Version = $version }
+  return @{ PackageName = "Waterfox$dash$build" ; Title = "Waterfox $build" ; URL64 = ($url -replace ' ','%20'); Version = $version }
```

Comment:
You will need to do some more special handling here.
The package name for the **current** edition of Waterfox should be `Waterfox` (yes, with uppercase W), but for the **classic** edition, it should be `waterfox-classic` all in lowercase characters.

Reply:
fixed